cardano-node/src/Cardano/Node/Tracing/Render.hs

```diff
@@ -0,0 +1,168 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+
+module Cardano.Node.Tracing.Render
+  (
+  -- renderBlockOrEBB
+       renderChunkNo
+     , renderHeaderHash
+     , renderHeaderHashForDetails
+     , renderChainHash
+     , renderTipBlockNo
+     , renderTipHash
+     , condenseT
+     , showT
+     , renderPoint
+     , renderPointAsPhrase
+     , renderPointForDetails
+     , renderRealPoint
+     , renderRealPointAsPhrase
+     , renderSlotNo
+     , renderTip
+     , renderTipForDetails
+     , renderTxId
+     , renderTxIdForDetails
+     , renderWithOrigin
+  ) where
+
+import           Cardano.Prelude
+import           Prelude (id)
+
+import qualified Data.ByteString.Base16 as B16
+import qualified Data.Text as Text
+import qualified Data.Text.Encoding as Text
+
+import           Cardano.Logging
+import           Cardano.Node.Queries (ConvertTxId (..))
+import           Cardano.Slotting.Slot (SlotNo (..), WithOrigin (..))
+import           Ouroboros.Consensus.Block (BlockNo (..), ConvertRawHash (..), RealPoint (..))
+import           Ouroboros.Consensus.Block.Abstract (Point (..))
+import           Ouroboros.Consensus.Ledger.SupportsMempool (GenTx, TxId)
+import qualified Ouroboros.Consensus.Storage.ImmutableDB.API as ImmDB
+import           Ouroboros.Consensus.Storage.ImmutableDB.Chunks.Internal (ChunkNo (..))
+import           Ouroboros.Consensus.Util.Condense (Condense, condense)
+import           Ouroboros.Network.Block (ChainHash (..), HeaderHash, StandardHash, Tip,
+                   getTipPoint)
+
+condenseT :: Condense a => a -> Text
+condenseT = Text.pack . condense
+
+-- renderBlockOrEBB :: BlockOrEBB -> Text
+-- renderBlockOrEBB (Block slotNo) = "Block at " <> renderSlotNo slotNo
+-- renderBlockOrEBB (EBB epochNo) = "Epoch boundary block at " <> renderEpochNo epochNo
+
+renderChunkNo :: ChunkNo -> Text
+renderChunkNo = Text.pack . show . unChunkNo
+
+-- renderEpochNo :: EpochNo -> Text
```

Comment:
Why is this commented out?

Reply:
Commented out because it was unused. Removed it completely